main.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Ok, you're going to want to change strategies here before moving on. You should generate a data model of your boards. Just like we did in the "JavaScript DOM Creation" exercise.